### ISSUE TITLE
License mess

### ISSUE THREAD

**Issue by AnHeuermann** [AMIT-HSBI/NonLinearSystemNeuralNetworkFMU.jl#10]
I want to use the MIT license for NonLinearSystemNeuralNetworkFMU.jl.

The FMI headers are licensed under BSD 2-Clause license. This is fine as long FMI-Standard-2.0.3/LICENSE.txt stays where it is.

OpenModelica source files, including generated files in the FMU/sources directory are licensed under [OSMC Public License ](https://www.openmodelica.org/developersresources/contributor-license-agreement?id=56:full-license) and [GPL v3](http://www.gnu.org/copyleft/gpl.html).
I'm changing files inside the FMU, so those changes need to be published under GPL v3 as well.
So basically the generated FMU is licensed under GPL v3.

In src/templates/special_interface.tpl.c I'm using types and functions from OpenModelica. That is part of NonLinearSystemNeuralNetworkFMU.jl, so I guess I need to make it GPL v3 as well.
